doc/API.md

```diff
@@ -638,11 +638,33 @@ returns the current settings along with metrics on the renter's spending.
     }
   },
   "financialmetrics": {
-    "contractspending": "1234", // hastings
-    "downloadspending": "5678", // hastings
-    "storagespending":  "1234", // hastings
-    "uploadspending":   "5678"  // hastings
-  }
+    "contractspending":     "1234", // hastings
+    "downloadspending":     "5678", // hastings
+    "storagespending":      "1234", // hastings
+    "uploadspending":       "5678"  // hastings
+  },
+  "contractmetrics": [
+    {
+      "allowance": {
+        "funds":       "1234", // hastings
+        "hosts":       24,
+        "period":      6048, // blocks
+        "renewwindow": 3024  // blocks
+      },
```

Comment:
This should be removed, the allowance is going to get more complicated and I think it will end up polluting the contract metrics. (e.g., allowance can have host blacklists or whitelists or ip ranges or emphasis on performance, etc)